mcwin32/Makefile.in

```diff
@@ -789,7 +789,7 @@ directories:			$(DIRECTORIES)
 artifacts:			buildinfo BUILDNUMBER
 
 %/.created:
-		-@$(PERL) ./support/mkdir_p.pl $(@D)
+		@$(PERL) $(top_builddir)/support/mkdir_p.pl $(@D)
```

Comment:
I'm hesitant to rewrite all utilities used by the makefiles in Perl. `mkdir` is somewhat special because such implementation exists already and because `mkdir` exists both in Windows shell and in POSIX.